blobfuse2-nightly.yaml

```diff
@@ -888,4 +888,85 @@ jobs:
               sudo rm -rf [0-9]
             displayName: 'Clean Agent Directories'
             condition: always() 
-  
\ No newline at end of file
+  - job: Set_8
+    timeoutInMinutes: 30
+
+    strategy:
+      matrix:
+        Ubuntu-18-CSI:
+          imageName: 'ubuntu-18.04'
+          containerName: 'test-cnt-ubn-18'
+          adlsSas: $(UBUNTU-18-ADLS-SAS)
+          hostedAgent: true
+          stressParallel: 3
+          fuselib: 'libfuse-dev'
+          tags: 'fuse2'
+    pool:
+      vmImage: $(imageName)
+
+    variables:
+      - group: NightlyBlobFuse
+      - name: ROOT_DIR
+        value: '$(System.DefaultWorkingDirectory)' 
+      - name: WORK_DIR
+        value: '$(System.DefaultWorkingDirectory)/azure-storage-fuse'
+
+    steps:
+      - checkout: none
+      - script: |
+          git clone https://github.com/Azure/azure-storage-fuse
+        displayName: 'Checkout Code & List commits'
+        workingDirectory: $(root_dir)
+          
+      # list commits from past 12hrs
+      - script: |
+          git checkout `echo $(Build.SourceBranch) | cut -d "/" -f 1,2 --complement`
+          git --no-pager log --since="12 hours ago" --stat
+        displayName: 'List Commits'
+        workingDirectory: $(work_dir)
+      
+      # install dependencies required for compiling blobfuse
+      - script: |
+          sudo apt-get update --fix-missing
+          sudo apt-get install ruby-dev build-essential $(fuselib) -y
+          sudo gem install fpm -V
+        displayName: "Installing Dependencies"
+
+      # build blobfuse2 and generate binary
+      - template: 'azure-pipeline-templates/build-release.yml'
+        parameters:
+          work_dir: $(work_dir)
+          root_dir: $(root_dir)
+          unit_test: false
+          tags: $(tags)
+
+      # place the generated binary files & any additional files in appropriate locations
+      - script: |
+          mkdir -p pkgDir/usr/bin/
+          mkdir -p pkgDir/usr/share/blobfuse2/
+          cp azure-storage-fuse/blobfuse2 pkgDir/usr/bin/blobfuse2
+          cp azure-storage-fuse/setup/baseConfig.yaml pkgDir/usr/share/blobfuse2/
+          cp azure-storage-fuse/sampleFileCacheConfig.yaml pkgDir/usr/share/blobfuse2/
+          cp azure-storage-fuse/sampleStreamingConfig.yaml pkgDir/usr/share/blobfuse2/
+          cp azure-storage-fuse/tools/postinstall.sh pkgDir/usr/share/blobfuse2/
+          mkdir -p pkgDir/etc/rsyslog.d
+          mkdir -p pkgDir/etc/logrotate.d
+          cp azure-storage-fuse/setup/11-blobfuse2.conf pkgDir/etc/rsyslog.d
```

Comment:
Does this work at install time without sudo for /etc/rsyslog.d

Reply:
You still have to use sudo but this works on the Docker container